tailwind.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
       sm: '576px',
       md: '768px',
       lg: '1024px',
+      lg2: '1124px',
```

Comment:
Could You please try not to make another breakpoint ?

Reply:
I explained it on dc